sources/packages/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -144,8 +144,7 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
         scholasticStandingData.exceptionalCosts ??
         existingOffering.exceptionalExpenses;
 
-      // TODO: When new offering type is added for report a change/ scholastic standing, update this.
-      offering.offeringType = OfferingTypes.applicationSpecific;
+      offering.offeringType = OfferingTypes.ScholasticStanding;
 
```

Comment:
is this supposed to be ScholastingStanding or scholastingStanding  ?

Reply:
Naming conventions for enum names is pascal case as per our agreement.

![image](https://user-images.githubusercontent.com/54600590/165820052-22af940a-3382-4f0f-aaef-ec770bd7656a.png)